blockchain/types/accountkey/account_key_weighted_multi_sig.go

```diff
@@ -76,6 +76,11 @@ func (a *AccountKeyWeightedMultiSig) Equal(b AccountKey) bool {
 func (a *AccountKeyWeightedMultiSig) Validate(r RoleType, pubkeys []*ecdsa.PublicKey) bool {
 	weightedSum := uint(0)
 
+	// To prohibit too much signatures
+	if len(a.Keys) < len(pubkeys) {
+		return false
```

Comment:
After this PR, too many signatures error condition is added to `invalid signature`. 
So why don't you return specific error in this function.

Reply:
Thank you for the suggestion. I will consider it later.

Reply:
Got it. :) Let's go~!!